src/Expensify.tsx

```diff
@@ -46,72 +45,46 @@ Onyx.registerLogger(({level, message}) => {
     }
 });
 
-const propTypes = {
-    /* Onyx Props */
-
-    /** Session info for the currently logged in user. */
-    session: PropTypes.shape({
-        /** Currently logged in user authToken */
-        authToken: PropTypes.string,
-
-        /** Currently logged in user accountID */
-        accountID: PropTypes.number,
-    }),
-
-    /** Whether a new update is available and ready to install. */
-    updateAvailable: PropTypes.bool,
-
-    /** Tells us if the sidebar has rendered */
-    isSidebarLoaded: PropTypes.bool,
-
-    /** Information about a screen share call requested by a GuidesPlus agent */
-    screenShareRequest: PropTypes.shape({
-        /** Access token required to join a screen share room, generated by the backend */
-        accessToken: PropTypes.string,
-
-        /** Name of the screen share room to join */
-        roomName: PropTypes.string,
-    }),
-
-    /** Whether the app is waiting for the server's response to determine if a room is public */
-    isCheckingPublicRoom: PropTypes.bool,
-
-    /** Whether we should display the notification alerting the user that focus mode has been auto-enabled */
-    focusModeNotification: PropTypes.bool,
-
-    ...withLocalizePropTypes,
+type ExpensifyOnyxProps = {
+    isCheckingPublicRoom: OnyxEntry<boolean>;
+    session: OnyxEntry<Session>;
+    updateAvailable: OnyxEntry<boolean>;
+    isSidebarLoaded: OnyxEntry<boolean>;
+    screenShareRequest: OnyxEntry<ScreenShareRequest>;
+    focusModeNotification: OnyxEntry<boolean>;
 };
 
-const defaultProps = {
-    session: {
-        authToken: null,
-        accountID: null,
-    },
-    updateAvailable: false,
-    isSidebarLoaded: false,
-    screenShareRequest: null,
-    isCheckingPublicRoom: true,
-    focusModeNotification: false,
-};
+type ExpensifyProps = ExpensifyOnyxProps;
 
 const SplashScreenHiddenContext = React.createContext({});
 
-function Expensify(props) {
-    const appStateChangeListener = useRef(null);
+function Expensify({
+    isCheckingPublicRoom = true,
+    session = {
+        authToken: undefined,
+        accountID: undefined,
+    },
```

Comment:
```suggestion
```

Reply:
If `session` can be `null` because it's an Onyx value and we are just setting `authToken` and `accountID` to `undefined` here, it looks to me that this default object is useless. Could you try to remove it and test if it works correctly?